### PR TITLE
3.3 fuck you puck and especially glitch

### PR DESCRIPTION
**Pull request by mstonefield**
All files that are needed for 3.3 and then some

### CODE REVIEW

File: common/pop_faction_types/00_xenoist.txt [hunk 1]
```diff
@@ -0,0 +1,756 @@
+##############################
```
Comment (cemal-efe-g) [inline common/pop_faction_types/00_xenoist.txt:1]: Tex plz remove this file, +100 influence for asari

File: common/scripted_effects/btr_pop_effects.txt [hunk 2]
```diff
@@ -12,11 +12,12 @@ generate_asariPop = {
...
 			#homeworld = THIS 
 			traits = {
-				trait = "mec_asari_trait_core"
-				trait = "mec_asari_trait_biotic"
-				trait = "trait_charismatic"
-				trait = "mec_asari_trait_sophisticated"
-				ideal_planet_class = "pc_continental"
+				trait = mec_asari_trait_core
+				trait = mec_asari_trait_biotic
+				trait = btr_asari_trait_smooth_talk
```
Comment (cemal-efe-g) [inline common/scripted_effects/btr_pop_effects.txt:17]: This pushes asari in!

File: common/solar_system_initializers/canon/01_canon_asari_republics_initializer.txt [hunk 3]
```diff
@@ -79,11 +79,12 @@ sys_parnitha_init = {
 					portrait = mec_asari
 					#homeworld = THIS 
 					traits = { 
-				  trait = "mec_asari_trait_core"
-				  trait = "mec_asari_trait_biotic"
-				  trait = "trait_charismatic"
-				  trait = "mec_asari_trait_sophisticated"
-					  ideal_planet_class = "pc_continental"
+				  		trait = mec_asari_trait_core
```
Comment (cemal-efe-g) [inline common/solar_system_initializers/canon/01_canon_asari_republics_initializer.txt:82]: Again this gets asari in!

File: common/on_actions/00_on_actions.txt [hunk 4]
```diff
@@ -2205,10 +2204,13 @@ on_leader_spawned = {
 		syndaw.1050 # Machine Intelligence ruler gets trait
 		distar.173 # New Leaders Get Brainslug Trait
 		clones.24 # New Leaders gain Clone Army Admiral traits
+		btr_asrep.03 # New Leaders gain Asari Trait
```
Comment (cemal-efe-g) [inline common/on_actions/00_on_actions.txt:2207]: Asari Stuff!!!

File: common/on_actions/btr_on_actions.txt [hunk 5]
```diff
@@ -27,12 +27,18 @@ on_game_start_country = {
...
+		btr_setup.110
 		btr_setup.2
 		btr_setup.201
 		btr_setup.202
 		btr_setup.203
 		btr_setup.3
 		btr_setup.4
+
+		### Game Start Events
+		btr_game_start.1 # Setup factions for each empire
```
Comment (cemal-efe-g) [inline common/on_actions/btr_on_actions.txt:39]: Again asari stuff!

File: prescripted_countries/btr_prescripted_countries.txt [hunk 6]
```diff
@@ -17,20 +17,20 @@ mec_asarirepublics_civ={
...
 
 	ethic = "ethic_fanatic_xenophile"
 	ethic = "ethic_egalitarian"
 
-	trait = "mec_asari_trait_core"
-	trait = "mec_asari_trait_biotic"
-	trait = "trait_charismatic"
-	trait = "mec_asari_trait_sophisticated"
-	trait = "trait_asari_habitability_preference"
+	trait = mec_asari_trait_core
```
Comment (cemal-efe-g) [inline prescripted_countries/btr_prescripted_countries.txt:27]: Asari stuff!

**Review by cemal-efe-g — CHANGES_REQUESTED** [review]

**Review by puckhead73 — APPROVED** [review]
Approved pending Glitch's comments

**Review by cemal-efe-g — APPROVED** [review]

**Pull request merged**